src/PointwiseFunctions/Hydro/Units.hpp

```diff
@@ -78,5 +80,36 @@ constexpr double pressure_unit =
  */
 constexpr double gauss_unit = 2.3558985e+19;
 }  // namespace cgs
+/*!
+* \brief Nuclear quantities given in terms of geometric units G = c = M_odot = 1
+*
+* Note that numbers may be quoted beyond (or truncated below) known
+uncertainties.
+*  Uncertainties  arise not only in the quantities themselves but also due to
+the poorly known
+* value of the suns mass, which appears in calculations of particle masses.
+
+*/
+namespace nuclear {
+/// Pressure in geometric units of 1 MeV/fm^3
+constexpr double MeV_per_fm_cubed = 2.885900818968523e-06;
+/// The unit of mass density in geometric units
+constexpr double neutron_mass_per_fm_cubed = 0.002711492496730566;
+/// Neutron mass expressed expressed in terms of neutrn mass density and
+/// fm expressed in terms of cm.
+constexpr double neutron_mass =
+    neutron_mass_per_fm_cubed * cube(1.0 / cgs::length_unit * 1.0e-13);
+/// The rest mass density of nuclear matter at saturation point, in geometric
+/// units. This value is based on a fiducial value of 2.8 g/cm^3, which is
+/// standard in astrophysics and consistent with modern nuclear theoretic
+/// estimates (see, e.g.
+/// https://journals.aps.org/prc/abstract/10.1103/PhysRevC.102.044321)
+constexpr double default_saturation_rest_mass_density = 4.5e-4;
+/// The default baryon mass in geometric units.
+/// Accepted value of atomic mass unit as expressed in
+/// solar masses
+/// Limited by precision of knowledge of solar mass.
+constexpr double default_baryon_mass = 8.34819e-58;
+}  // namespace nuclear
 }  // namespace units
```

Comment:
It's not actually the solar mass that has limited precision: G M_sun is known to something like 10 digits. It's G that has limited precision, which makes the conversion to kg problematic. The only reason this might matter is when it comes to comparing different codes, or different versions of spectre where someone "updates" the units with improved values. The best way to handle this would be to define G as a constant in one place and use it explicitly where needed for conversions. Then updates would be clear. I'm happy not to do this in this PR, but then perhaps you should file this comment as an Issue.

Reply:
I agree; my main point with this comment is to point out that the baryon mass is actually known to much higher precision than is represented here.  I'd be happy to give numbers to highest precision in the units they're known best, and then report the uncertainty in G as the limiting factor in the conversion.